### ISSUE TITLE
diferent color on download progress based on status of the torrent

### ISSUE THREAD

**Issue by SmokeyBR** [Novik/ruTorrent#2616]
### Is your feature request related to a problem?

Not really.... more a personal preference i find it hard or not really in your face to look for status of a torrent if its seeding or not

### Describe the solution you'd like

would like to check if theres any interest in making a plugin or css (whichever would be better and possible) that will colorize the download bar progress into different colors based on their status to me seems much better to identify the status of the torrent.

### Additional context

something like this
![status](https://github.com/Novik/ruTorrent/assets/19333582/d83a31c3-636f-44f7-910b-8fef927b018e)

this is really a hacky plugin that i came up with using chatgpt it does not work well, also i hijacked this [plugin](https://github.com/Gyran/rutorrent-ratiocolor) to make it work ... kindof

**Comment by stickz** [comment]
@SmokeyBR Yes, we can change the ratio and download progress bar colours on ruTorrent.
Please feel free to open a pull request and submit it to the develop branch for `v4.3`. 

## A few notes
1.  It will need to be a plugin so it can be disabled.

2.  We don't want to square off and fill the torrent ratio by default. Optional feature disabled by default in the plugin `conf.php` is fine. Pretty much anything flies here if it not enabled by default. 

3. We can change the ratio colours by default, but we need to be conservative with our colour selection and allow the user to configure the feature. A rainbow of colours would not be appropriate for a ruTorrent default configuration.

4. A sub menu in the settings menu (where the user can configure the plugin) is strongly recommended. We would like the user to be able to configure which status evokes a colour change. We are okay with `checking` and `seeding` enabled by default.
